### ISSUE TITLE
Missing property schema for properties that start with 'is[A-Z]'

### ISSUE THREAD

**Issue by ReverendDread** [SMILEY4/ktor-openapi-tools#60]
If you start a property name with 'is' and use a capital letter directly following, the property schema fails to generate.

Examples:
```
data class Irrelevant(
    val isNamed: String,
    val isIrrelevent: String
)
```

**Comment by SMILEY4** [comment]
Hi,  this is an problem with the underlying schema-generation-library i am using (see https://github.com/victools/jsonschema-generator/issues/363), so i don't think there is much i can do on our side. However, the linked issue has already been resolved and fixed, just not yet released. 
My plan would be to wait for the new version of jsonschema-generator and upgrade as soon as possible.

**Comment by Sterta** [comment]
@SMILEY4 Release is ready https://github.com/victools/jsonschema-generator/releases/tag/v4.32.0 :)

**Comment by SMILEY4** [comment]
Now fixed with version 2.7.0
